src/components/Tile.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Tile({children}) {
```

Comment:
Deze Tile functie zou de props mee moeten krijgen ( img, title, text ). Daarbij zou je iets van een if statement kunnen gebruiken om te checken of er een img is meegegeven of niet en zodoende als img-tile moet werken of als text.